oeps/oep-0017-bp-feature-toggles.rst

```diff
@@ -44,7 +44,7 @@ Given the many benefits of using feature toggles, edX development teams have bee
 
 Aligning on a common best practice for feature toggling is critical for both the short-term and long-term health of the system. Using different standards, strategies, testing procedures, etc., leads to confusion, production failures, and long-term maintenance issues.
 
-.. _long-term feature branches: https://blog.newrelic.com/2012/11/14/long-running-branches-considered-harmful/
+.. _long-term feature branches: https://newrelic.com/blog/best-practices/long-running-branches-considered-harmful
```

Comment:
This means by default it is checking all links on build, or did you manually run a separate check? Is it a problem that our builds could fail when a link is temporarily unavailable?

Reply:
I did the link checking manually as an added step, it will not run by default.  We may want to add it in the future but as I noted in the commit, link checking can be unreliable and so it might just be a one-off things we run sometimes.

Reply:
Thanks. I hadn't read the commit messages.